sisl/geom/bilayer.py

```diff
@@ -0,0 +1,95 @@
+import numpy as np
+
+from sisl import Atom, geom, Cuboid
+
+__all__ = ['bilayer']
+
+
+def bilayer(bond, atom=None, twist=(5, 6), separation=3.35, return_angle=False, layer='both'):
+    """ Commensurate unit cell of a twisted (hexagonal) bilayer structure.
+
+    This routine follows the prescription of twisted bilayer graphene found in
+    Laissardiere et al., Nano Lett. 10, 804-808 (2010).
+
+
+    Parameters
+    ----------
+    bond : float
+       length between atoms in nano-tube
+    atom : Atom(6)
+       bilayer atoms
+    twist : (int, int)
+       coordinates (m, n) defining the twist angle (theta)
+    separation : float
+       distance between the two layers
+    return_angle : bool
+       computed twist angle is returned in addition to the geometry instance
+    layer : {'both', 'bottom', 'top'}
+       control which layer(s) to include in geometry
+    """
+    if atom is None:
+        atom = Atom(Z=6, R=bond * 1.01)
+
+    # Compute twist angle
+    m, n = twist
+    if m > n:
+        m, n = n, m
+
+    if not (isinstance(n, int) and isinstance(m, int)):
+        raise RuntimeError('twist coordinates need to be integers!')
+
+    cos_theta = (n ** 2 + 4 * n * m + m ** 2) / (2 * (n ** 2 + n * m + m ** 2) )
+    theta = np.arccos(cos_theta) * 360 / (2 * np.pi)
+
+    # Build structure
+    graphene = geom.graphene(bond=bond, atom=atom)
```

Comment:
May I suggest doing `geom.honeycomb` to clarify that this is not merely graphene, but potentially also hBN?
And then say `honeycomb = geom.honeycomb` for further clarification.

Reply:
Good idea. Now used in f5695c859bb205eae43694de62b3b4433b435b9e